### ISSUE TITLE
Do occupation and industry belong in Organization?

### ISSUE THREAD

**Issue by DMSaunders** [jobdataexchange/Data-Modeling#29]
`occupationalCategory` is currently in `Organization`. Should it be in `JobPosting` since an org almost always has multiple different occupations? For that matter, what happens when an org has multiple industries?

**Comment by DMSaunders** [comment]
Here is the description of industryCode "In a JDX job description, there my be multiple levels and locations for an organization identified and each may have different industry codes based on the kinds of work performed at those locations; the NAICS industry code here in JDX applies SOLELY to the location where the work of the job being described will be performed."

**Comment by stuartasutton** [comment]
Danielle, the title of the issue includes both occupationCategory and IndustryCode. No, occupationCategory does not belong on Organization; but industryCode does but was inadvertently left off. I have updated github full and releases.

**Comment by stuartasutton** [comment]
Didn’t answer the question of an organization having multiple industry codes. First, nothing precludes an organization from having multiple industryCodes...BUT, it is not the just any of the codes an organization might have, but only those relevant to the specific location of employment.

**Comment by DMSaunders** [comment]
Since each `JobPosting` has its own `jobLocation`, which makes sense because we expect a job posting to have a location where that specific job will be performed, AND each location has its own `industryCode`, then it has to follow that each `JobPosting` should have its own `industryCode`, right? Otherwise an `industryCode` property in Organization would have to list of all the industryCodes that have ever been used in conjunction with that Organization. This is based on my understanding that an Organization can be reused in multiple JobPostings because its properties pertain to the *entire* org, correct? So under Organization, the properties `email`, `address`, etc should be for the headquarters, not the specific job (unless they happen to be the same).

By the way, will we be following this guidance from Google JobPosting? 

> If the job has multiple locations, add multiple jobLocation properties in an array. Google will choose the best location to display based on the job seeker's query.

**Comment by stuartasutton** [comment]
No, I don't think that results in the JobPosting having the industry code. The code belongs to an org at the specific location where the job is performed. So a case can be made that IndustryCode is a characteristic of the location (which could have multiple such codes).  But is it not a characteristic of the posting. This conversation needs Bob's input and he is not on github.

I see nothing in the JDX schema precluding following the Google guidance.
